auto-run-nondex/runNondex.sh

```diff
@@ -16,7 +16,7 @@ runNondex () {
     input="modnames"
     while IFS= read -r line
     do
-        mvn edu.illinois:nondex-maven-plugin:$nondex_version:nondex -pl :$line -Dlicense.skip=true | tee ./.runNondex/LOGSSS/$line.log
+        mvn edu.illinois:nondex-maven-plugin:$nondex_version:nondex -pl :$line -Dlicense.skip=true -Drat.skip=true -DlicenseCheck.numUnapprovedLicenses=99999 -fae | tee ./.runNondex/LOGSSS/$line.log
```

Comment:
1. Do the flags require explicit `=true`?

2. Which example needed that `numUnapprovedLicenses`?

3. Does the rest of script support the case where multiple modules have a test failure? (Maybe also expand `-fae` to the full name, so it's easier to read/maintain.)

Reply:
1. Do the flags require `explicit =true`?

The explicit `true` should not be required, but improves readability by making the purpose of the flags `-Dlicense.skip=true` `-Drat.skip=true` as boolean flags

2. Which example needed that `numUnapprovedLicenses`?

I don't know, but is a nice to have as a  workaround for a project with many dependencies that have unapproved licenses.

3. Does the rest of script support the case where multiple modules have a test failure? (Maybe also expand -fae to the full name, so it's easier to read/maintain.)

Changing -fae to --fail-at-end might be a good idea for readability